datalake/src/main/java/com/sequenceiq/datalake/flow/detach/SdxDetachActions.java

```diff
@@ -186,22 +217,19 @@ protected Object getFailurePayload(SdxEvent payload, Optional<SdxContext> flowCo
     @Bean(name = "SDX_DETACH_EXTERNAL_DB_FAILED_STATE")
     public Action<?, ?> sdxDetachExternalDBFailedAction() {
         return new AbstractSdxAction<>(SdxDetachFailedEvent.class) {
-            @Override
-            protected SdxContext createFlowContext(FlowParameters flowParameters,
-                    StateContext<FlowState, FlowEvent> stateContext,
-                    SdxDetachFailedEvent payload) {
-                return SdxContext.from(flowParameters, payload);
-            }
-
```

Comment:
why are these cod blocked removed?

Reply:
Recently the parent class, `AbstractSdxAction`, has been given the new method:

```
protected SdxContext createFlowContext(FlowParameters flowParameters, StateContext<FlowState, FlowEvent> stateContext, P payload) {
        return SdxContext.from(flowParameters, payload);
    }
```

so all subclasses automatically have this method implemented in this way.

Therefore it's unnecessary for `SdxDetachActions` I removed it to make the class cleaner.